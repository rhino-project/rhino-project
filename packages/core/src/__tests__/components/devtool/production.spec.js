import { render } from '@testing-library/react';
import { afterEach, describe } from 'vitest';

vi.stubEnv('NODE_ENV', 'production');

describe('RhinoDevTool production', () => {
  afterEach(() => {
    vi.unstubAllEnvs();
  });

  it('should return null', async () => {
    const Devtool = await import('../../../components/devtool');
    const { asFragment } = render(<Devtool.RhinoDevTool />);
    expect(asFragment()).toMatchSnapshot();
  });
});
