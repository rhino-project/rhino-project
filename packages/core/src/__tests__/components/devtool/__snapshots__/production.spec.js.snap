// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`RhinoDevTool production > should return null 1`] = `<DocumentFragment />`;
